output/schema/validation-errors.json

```diff
@@ -830,6 +830,12 @@
       ],
       "response": []
     },
+    "indices.get_data_stream": {
+      "request": [
+        "Request: missing json spec query parameter 'verbose'"
+      ],
+      "response": []
+    },
```

Comment:
We need to add that flag to the specification: https://github.com/elastic/elasticsearch/pull/112303